ROMFS/px4fmu_common/init.d/rc.interface

```diff
@@ -223,6 +225,46 @@ then
 	fi
 fi
 
+# TODO: fix this mess
+echo "MIXER_AUX: ${MIXER_AUX}"
+echo "AUX_MODE: ${AUX_MODE}"
+echo "OUTPUT_CMD: ${OUTPUT_CMD}"
+echo "OUTPUT_MODE: ${OUTPUT_MODE}"
+echo "PWM_AUX_OUT: ${PWM_AUX_OUT}"
+echo "PWM_AUX_RATE: ${PWM_AUX_RATE}"
+echo "OUTPUT_DEV: ${OUTPUT_DEV}"
+echo "OUTPUT_AUX_DEV: ${OUTPUT_AUX_DEV}"
+
+if [ $MIXER_AUX != none -a $AUX_MODE = none -a -e $OUTPUT_AUX_DEV ]
+then
+	#
+	# Load aux mixer.
+	#
+	if [ -f ${SDCARD_MIXERS_PATH}/${MIXER_AUX}.aux.mix ]
+	then
+		set MIXER_AUX_FILE ${SDCARD_MIXERS_PATH}/${MIXER_AUX}.aux.mix
+	else
+
+		if [ -f /etc/mixers/${MIXER_AUX}.aux.mix ]
+		then
+			set MIXER_AUX_FILE /etc/mixers/${MIXER_AUX}.aux.mix
+		fi
+	fi
+
+	if mixer load ${OUTPUT_AUX_DEV} ${MIXER_AUX_FILE}
+	then
+		echo "INFO  [init] Mixer: ${MIXER_AUX_FILE} on ${OUTPUT_AUX_DEV}"
+
+		# Set PWM_AUX output frequency.
+		if [ $PWM_AUX_RATE != none ]
+		then
+			pwm rate -c ${PWM_AUX_OUT} -r ${PWM_AUX_RATE} -d ${OUTPUT_AUX_DEV}
+		fi
+	else
+		echo "ERROR  [init] Failed loading mixer: ${MIXER_AUX_FILE}"
+	fi
+fi
```

Comment:
I think this logic needs some good comments, it's not very comprehensible, and I can't tell if it's correct or not.